mascara/src/mascara.js

```diff
@@ -13,8 +14,11 @@ const provider = setupProvider({
 })
 instrumentForUserInteractionTriggers(provider)
 
-const web3 = new Web3(provider)
-setupDappAutoReload(web3, provider.publicConfigStore)
+// export metamask
+setupDappAutoReloadForMetaMask(provider, provider.publicConfigStore)
+// export web3
+const web3 = new Web3(global.metamask.createDefaultProvider())
```

Comment:
lets drop web3 for mascara
maybe ok to remove the dapp autoreload feature as the usage detection will need to be retooled